infra/ansible/scripts/bench_operational_verify.sh

```
@@ -202,6 +202,43 @@ else
+else
+  log_fail "driver tree empty"
+fi
+read_pv="$(api_post "/api/bacnet/read" "$(python3 -c 'import json; print(json.dumps({"device_instance":5007,"object_identifier":"analog-input,1168","property_identifier":"present-value"}))')")"
+if echo "$read_pv" | python3 -c 'import json,sys; d=json.load(sys.stdin); sys.exit(0 if "value" in d else 1)'; then
+  log_ok "On-demand read PV analog-input,1168"
+else
+  log_fail "read PV failed: ${read_pv}"
+fi
+
+log_info "Arrow rule lint (reject legacy evaluate / pandas)"
+for label code in "legacy:def evaluate(row,cfg): return True" "pandas:import pandas as pd"; do
```

Shellcheck (0.11.0)

[error] Couldn't parse this for loop. Fix to allow more checks.

(SC1073)